add-custom-dynamic-entity-data-source/src/main/java/com/esri/arcgismaps/sample/addcustomdynamicentitydatasource/screens/MainScreen.kt

```diff
@@ -74,7 +77,28 @@ fun MainScreen(sampleName: String) {
                         .weight(1f),
                     mapViewProxy = mapViewModel.mapViewProxy,
                     arcGISMap = mapViewModel.arcGISMap,
-                    onSingleTapConfirmed = mapViewModel::identify
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    content = {
+                        mapViewModel.selectedGeoElement?.let { selectedGeoElement ->
+                            Callout(
+                                modifier = Modifier.wrapContentSize(),
+                                geoElement = selectedGeoElement,
+                                // Optional parameters to customize the callout appearance.
+                                shapes = CalloutDefaults.shapes(
+                                    calloutContentPadding = PaddingValues(
+                                        4.dp
+                                    )
+                                ),
+                                colorScheme = CalloutDefaults.colors(MaterialTheme.colorScheme.background, MaterialTheme.colorScheme.primary)
+                            ) {
+                                Column(Modifier.padding(4.dp)) {
+                                    Text(
+                                        text = mapViewModel.observationString
+                                    )
```

Comment:
The default text size seems to be too large for the Callout content, maybe using a smaller typography here could be helpful: 
```suggestion
                                    Text(
                                        text = mapViewModel.observationString,
                                        style = MaterialTheme.typography.labelSmall
                                    )
```